src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/Order.orm.xml

```diff
@@ -20,17 +20,19 @@
             <join-column name="channel_id" referenced-column-name="id" nullable="true" />
         </many-to-one>
 
-        <many-to-one field="shippingAddress" target-entity="Sylius\Component\Addressing\Model\AddressInterface">
+        <many-to-one field="shippingAddress" target-entity="Sylius\Component\Addressing\Model\AddressInterface" orphan-removal="true">
```

Comment:
This is unidirectional relation, are we sure that orphan removal works here at all?